inc/CalculatorStrings.h

```diff
@@ -16,16 +16,66 @@ namespace calculator
     struct CalculatorStrings
     {
         /**
-         * @brief prompt passed to user before retrieving input from them
+         * @brief ""
+         * 
+         */
+        static const std::string EMPTY_STRING;
```

Comment:
A struct is a class in C++ (mostly) and static const std::string in this way is subject to static order initialization fiasco see https://meowni.ca/posts/static-initializers/